packages/mobile/src/components/publish-content-drawer/PublishContentDrawer.tsx

```diff
@@ -71,13 +76,41 @@ const useStyles = makeStyles(({ palette, spacing }) => ({
   }
 }))
 
-export const PublishPlaylistDrawer = () => {
+export const PublishContentDrawer = () => {
   const dispatch = useDispatch()
-  const playlistId = useSelector(getPlaylistId)
   const neutral = useColor('neutral')
   const { toast } = useManualToast()
   const styles = useStyles()
-  const { isOpen, onClose } = useDrawerState('PublishPlaylistConfirmation')
+  const {
+    isOpen,
+    onClose,
+    data: { contentId, contentType }
+  } = usePublishContentModal()
+  const [dismissToastKey, setDismissToastKey] = useState<string | undefined>()
+  const currentTrack = useSelector((state) =>
+    getTrack(state, { id: contentId })
+  )
+
+  const previousIsPublishingTrack = usePrevious(currentTrack?._is_publishing)
+
+  // (Tracks only) - Check for changes to is_publishing to dismiss the toast
```

Comment:
comment still relevant?

Reply:
Yea still relevant, I explained it in the comment I posted myself; the playlist saga dismisses the toast, whereas for tracks I did it in-component